### PR TITLE
LPS-147479

### PR DESCRIPTION
**Pull request by eliasantosf**
Ticket: [LPS-147479](https://issues.liferay.com/browse/LPS-147479)
Story: [LPS-139394](https://issues.liferay.com/browse/LPS-139394)

### CODE REVIEW

**Review by naraandrade — CHANGES_REQUESTED** [review]
@eliasantosf Please, display the actions inside timers with its specificities.

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseAction.js [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+		label: Liferay.Language.get('on-entry'),
+		value: 'onEntry',
+	},
+	{
+		label: Liferay.Language.get('on-exit'),
+		value: 'onExit',
+	},
+];
+
+const BaseAction = ({
```
Comment (naraandrade) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseAction.js:34]: @eliasantosf Could you please rename the component to BaseActionsInfo? Thanks :)

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseAction.js [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+					type="number"
+					value={priority}
+				/>
+			</ClayForm.Group>
+			<div className="sheet-subtitle" />
+		</>
+	);
+};
+
+BaseAction.propTypes = {
```
Comment (naraandrade) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseAction.js:248]: @eliasantosf Please, update the PropTypes of this file and also of the other files like TimerAction, TimerFields, TimerDuration, TimerInfo etc.

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/timers/select-action/ActionTypeAction.js [hunk 3]
```diff
@@ -49,7 +51,7 @@ const ActionTypeAction = () => {
 							})
 						}
 					>
-						Add Button Placeholder
+						{Liferay.Language.get('add-script')}
```
Comment (naraandrade) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/timers/select-action/ActionTypeAction.js:57]: @eliasantosf Please, rename this button to "new-section"

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by cgmonte** [comment]
Just started reviewing :)

**Comment by cgmonte** [comment]
Just started reviewing :)

**Comment by cgmonte** [comment]
Just started reviewing :)

**Comment by cgmonte** [comment]
ci:test:sf

**Review by cgmonte — APPROVED** [review]

**Comment by cgmonte** [comment]
ci:test:relevant

**Review by cgmonte — APPROVED** [review]

**Comment by cgmonte** [comment]
ci:test:sf

**Comment by cgmonte** [comment]
Just started reviewing :)

**Comment by cgmonte** [comment]
ci:test:relevant

**Comment by cgmonte** [comment]
ci:forward

**Pull request closed without merging**